analyzer/src/main/kotlin/managers/Maven.kt

```diff
@@ -115,6 +117,12 @@ class Maven(
             scope.dependencies += parseDependency(node, packagesById)
         }
 
+        val declaredLicenses = MavenSupport.parseLicenses(mavenProject)
+
+        // See http://maven.apache.org/ref/3.6.3/maven-model/maven.html#project which says: "If multiple licenses are
+        // listed, it is assumed that the user can select any of them, not that they must accept all."
```

Comment:
While it's in the docs I wonder if this property is also interpreted by the community like this, or if there are many projects that declare multiple licenses but they are not a choice.

Reply:
That's a good question. From looking at existing POMs I also got the impression that the community at least sometimes wants to express `AND` instead of `OR`. But pragmatically, that's then their problem. The Maven docs are very clear here.

Reply:
I tend to agree, I'm just a little concerned because ORT should try to capture reality and if this feature is commonly misused ORT should account for that. However, I don't want to block that. Shall we have a quick discussion about this in the dev meeting tomorrow?

Reply:
Well, "capture reality" should rather say "capture facts", which means to not apply any interpretation. I acknowledge we have some places in ORT where we fixup common *syntactic* errors (where it's absolutely clear what the correction should be), but IMO we should not aim to fixup *semantic* errors like misuse / misinterpretation of package manager features.

I guess this can wait yet one more day to get merged, so if you'd like to talk about it in the dev meeting, please put it onto the agenda.